.travis.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   include:
     - stage: install
       name: "Install Dependencies"
-      script: yarn install --frozen-lockfile
```

Comment:
@ColdFire87 - any reason not to also use "frozen-lockfile" here to ensure it's not out of sync?